### ISSUE TITLE
MacOS Libindy setup

### ISSUE THREAD

**Issue by blu3beri** [hyperledger/aries-framework-javascript#326]
The current macOS guide for installing libindy fails as the openssl tap made by rbenv does not comply with the new architecture. 

A [pull request](https://github.com/rbenv/homebrew-tap/pull/2) has been made to the repository from rbenv that should fix it

**Comment by TimoGlastra** [comment]
Is this something we can fix over here, or do we just need to wait the PR to be merged?

**Comment by blu3beri** [comment]
If anyone has the same issues, installing openssl with [this file](https://raw.githubusercontent.com/felixbuenemann/homebrew-tap/e472b7861b49cc082d1db0f66f265368da107589/Formula/openssl%401.0.rb) works.

1. Download the file and name it openssl-patch.rb
2. Go to your downloads directory in the terminal
3. brew install ./openssl-patch.rb

**Comment by TimoGlastra** [comment]
@blu3beri is this fixed with https://github.com/hyperledger/aries-framework-javascript/pull/328?

**Comment by blu3beri** [comment]
unfortunately, the PR needs to be merged for that to work with Apple Silicon. Once that is merged there is another issue of homebrew moving the lib path to `/opt/homebrew/lib` instead of `/usr/local/lib`. Could be added somewhere as a symlink so that would not be such a big deal.

**Comment by TimoGlastra** [comment]
https://medium.com/@AnimoSolutions/how-to-install-libindy-for-macos-with-apple-silicon-to-use-with-aries-framework-javascript-ffe9719ab2e
